packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -6171,15 +6166,16 @@ class MultiChildRenderObjectElement extends RenderObjectElement {
   }
 
   @override
-  void mount(Element parent, dynamic newSlot) {
+  void mount(Element? parent, dynamic newSlot) {
     super.mount(parent, newSlot);
-    _children = List<Element>(widget.children.length);
-    Element previousChild;
-    for (int i = 0; i < _children.length; i += 1) {
+    final List<Element?> children = List<Element?>.filled(widget.children.length, null, growable: false);
+    Element? previousChild;
+    for (int i = 0; i < children.length; i += 1) {
       final Element newChild = inflateWidget(widget.children[i], IndexedSlot<Element>(i, previousChild));
-      _children[i] = newChild;
+      children[i] = newChild;
       previousChild = newChild;
     }
+    _children = children.cast<Element>();
```

Comment:
we have to avoid this cast

Reply:
you can use the same trick mentioned earlier with a `class _NullElement extends Element` with a static instance.

Reply:
in this case it might be possible to use List.generate.

Reply:
Done